python/lsst/daf/butler/remote_butler/server/handlers/_external.py

```diff
@@ -27,6 +27,8 @@
 
 from __future__ import annotations
 
+from lsst.daf.butler.remote_butler.server.handlers._utils import set_default_data_id
```

Comment:
I think at the time they were added, there was a theory that the Butler server code was supposed to move into a separate repository at some point.  Made them all relative since I don't think we want to do that anymore.